settings.php

```diff
@@ -0,0 +1,226 @@
+<?php
+/**
+ * Plugin configuration page
+ * Based on: https://vedovini.net/2015/10/using-the-wordpress-settings-api-with-network-admin-pages/
+ */
+
+add_action( 'init', array( 'Multisite_Enhancements_Settings', 'init' ) );
+
+class Multisite_Enhancements_Settings {
+
+	/**
+	 * Default options settings
+	 */
+	static protected $default_options = array(
+		'remove-logo'         => '1',
+		'add-favicon'         => '1',
+		'add-blog-id'         => '1',
+		'add-css'             => '1',
+		'add-plugin-list'     => '1',
+		'add-theme-list'      => '1',
+		'add-site-status'     => '1',
+		'add-ssl-identifier'  => '1',
+		'add-manage-comments' => '1',
+		'add-network-plugins' => '0', // not needed for WP >= 3.7
+		'add-new-plugin'      => '1',
+		'filtering-themes'    => '1',
+		'change-footer'       => '1',
+	);
+
+	/**
+	 * Init function to register all used hooks.
+	 */
+	public function __construct() {
+		// register settings
+		add_action( 'admin_init', array( $this, 'settings_init' ) );
+		// add menu item in the network Settings menu
+		add_action( 'network_admin_menu', array( $this, 'add_settings_menu' ) );
+		// register our custom URL to save the options
+		add_action( 'network_admin_edit_wpme_update_settings', array( $this, 'update_settings' ) );
+	}
+
+	/**
+	 * Initialize the class.
+	 */
+	public static function init() {
+		$class = __CLASS__;
+		if ( empty( $GLOBALS[ $class ] ) ) {
+			$GLOBALS[ $class ] = new $class();
+		}
+	}
+
+	/**
+	 * Register settings
+	 */
+	public function settings_init() {
+		if ( ! current_user_can( 'manage_network_options' ) ) {
+			return;
+		}
+
+		// register database option
+		register_setting(
+			'wpme_options',	// group name, used in settings_fields() call
+			'wpme_options'  // database option name
+		);
+
+		// register configuration page section
+		add_settings_section(
+			'wpme_general',		// unique ID
+			__( 'General configuration', 'multisite-enhancements' ),	// section title
```

Comment:
Each string for translation need a validation, so use here `esc_html__()`.

Reply:
Solved in https://github.com/bueltge/wordpress-multisite-enhancements/pull/63/commits/90be82b64d9ae7d91178f838977d57c038cd01b2

I added the escape function inside the output loop, as per codex recommendation that ["Output escaping should occur as late as possible."](https://developer.wordpress.org/themes/theme-security/data-sanitization-escaping/#escaping-securing-output)